src/polyhedralGravity/calculation/GravityEvaluable.cpp

```diff
@@ -66,7 +66,7 @@ namespace polyhedralGravity {
 
         //10. Step: Final expressions after application of the prefix (and a division by 2 for the potential)
         potential = (potential * prefix) / 2.0;
-        acceleration = acceleration * prefix;
+        acceleration = acceleration * (-1.0 * prefix);
```

Comment:
One thing I am mildly worried about is that once you push this if somebody built some code based on the old version and updates / reinstalls their env they will get wrong results without knowing about the change. It would be desirable to make their code break in that case. Maybe we could rename some function or something along the way to make this a breaking change that shows a message that the sign has changed if the they try to call the old version on this version?

Reply:
Overall, I agreed; however, I am unsure about how.
A `warning` or `rename/ depcreation` seems un-intuitive.

The best I could imagine would be something like a `verbose` option being `True` by default. The option could provide information about recent changes but also enable logging.
`evaluate(..., verbose=False)`.
So, basically, a new feature.

On the other hand, simply having more options also seems a bit counterintuitive and contradicts the "slim, easy-to-use interface idea."

Not sure. Anyway, I will merge this so that we can get the JOSS paper rolling.

The `verbose` option might be something that fits nicely with #34, which will adapt the signature.
For now, I hope that a user who notices a different behavior's first idea is to check the Changelog—and there, I will definitely put the change in bold letters (Also, the package description now states which convention we follow).